api/telemetry/v1alpha1/tenant_types.go

```diff
@@ -88,12 +88,15 @@ type TenantSpec struct {
 	SubscriptionNamespaceSelectors []metav1.LabelSelector `json:"subscriptionNamespaceSelectors,omitempty"`
 
 	// Determines the namespaces from which logs are collected by this tenant.
-	// If initialized with an empty list, logs from all namespaces are collected.
-	// If uninitialized, no logs are collected.
+	// Cannot be used together with SelectFromAllNamespaces.
```

Comment:
can we add a CEL validator for this?

Reply:
This was a leftover from the previous PR: #115.
Sure we can!